packages/legend-studio/src/stores/EditorStore.ts

```diff
@@ -1281,6 +1281,12 @@ export class EditorStore {
         showLoading: true,
       });
       try {
+        if (this.isInViewerMode) {
+          this.graphState.buildGraph(
```

Comment:
How does this fix the problem. 
I would assume that the `buildGraph` would still break with the given entities and form an incomplete graph and then give incomplete text.